src/UI/examples/Toast/Standard/with_delay_time.php

```diff
@@ -0,0 +1,13 @@
+<?php declare(strict_types=1);
+
+namespace ILIAS\UI\examples\Toast\Standard;
+
+function with_delay_time()
```

Comment:
If possible, please define the return type of this function (here or in the related PR).

Reply:
Resolved in dependend PR https://github.com/ILIAS-eLearning/ILIAS/pull/3671